### ISSUE TITLE
Vets-api terminal access for Matthew Christianson

### ISSUE THREAD

**Issue by mchristiansonVA** [department-of-veterans-affairs/va.gov-team#93178]
### Your Name

Matthew Christianson

### Your GitHub handle (terminal access is managed through GitHub)

mchristiansonVA

### Your dsvagovcloud AWS user name (if applicable)

matthew.christianson

### Team, Role, and Company of the target individual

Team Dash, Software Engineer, Ad Hoc LLC

### Product Manager (PM) name and email

Kayla Watanabe

### Product Owner (PO) name and email

Janet Coutinho

### COR Name (leave "None" if you listed an AWS user name above)

None

### Vendor Onboarding Representative (VOR) Name (leave "None" if you listed an AWS user name above)

None

### Please confirm if you are listed as a team member in Atlas or are listed in Platform Team Roster

- [X] [Team Member in Atlas](https://www.va.gov/atlas/product_directory/team_members)
- [X] [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)

### Please confirm E-QIP Transmittal/Adjudication. This could be a screenshot of your Transmittal/Adjudication email, a link to another request where you've submitted this proof (i.e. AWS/SOCKS), etc.

- [X] I have attached proof that I have either transmitted my E-QIP survey or previously received a favorable decision (by including in "Additional Notes" section below)

### ArgoCD terminal access requested for the following environments

- [X] dev
- [X] staging
- [X] sandbox
- [X] production (if requesting, justify prod access in 'Additional Notes' section)

### Additional Notes

I need production access for troubleshooting issues with our new 526 Disability Compensation Claims service that VA.gov just launched.

![image](https://github.com/user-attachments/assets/c162986e-7c50-4150-b1c3-e4222eadf217)

### Additional resources for Support Engineer

- [ ] For users on a VFS team but not listed as a team member in Atlas, they must start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] Instructions for handling requests [here](https://vfs.atlassian.net/wiki/spaces/PTST/pages/2183037084/Backend+Support+Responsibilities#Handle-vets-api-ArgoCD-terminal-access) - Support Only

**Comment by AnthonyWFlowers** [comment]
@mchristiansonVA  You’ve been granted access to the `dev`, `staging`, and `sandbox` environments.

Access to `prod` is pending approval by OCTODE, and we'll update you as soon as it's available.

**Comment by JonathanKamensDVA** [comment]
Approved.

**Comment by AnthonyWFlowers** [comment]
Greetings @mchristiansonVA You've been granted access to Production, with access to Production limited to 6 months, expiring 03/20/25.

Please set a reminder for yourself to return to this Issue and request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 72 hour window to respond. Failure to respond will result in your access being revoked and will require you to fill out a new request.

**Comment by AnthonyWFlowers** [comment]
Link to reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1726851068621379).

**Comment by AnthonyWFlowers** [comment]
@mchristiansonVA  Your Vets API Terminal Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your access will be revoked and you will need to fill out a new request.

**Comment by mchristiansonVA** [comment]
@AnthonyWFlowers Yes please, I'd like to request an extension of my Vets API Terminal production access. Thanks!

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by bdech** [comment]
@mchristiansonVA, your Production access has been extended for 6 months, expiring 9/20/25.

Please set a reminder for yourself to return to this Issue, reopen it, and request an extension within 1 week of the expiration date. You'll receive 1 reminder with a 48 hour window to respond. Failure to respond will result in your access being revoked and will require you to fill out a new request.

**Comment by bdech** [comment]
support slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1743017411720299)

**Comment by AnthonyWFlowers** [comment]
@mchristiansonVA  Your Vets API Terminal Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your access will be revoked and you will need to fill out a new request.

**Comment by AnthonyWFlowers** [comment]
@mchristiansonVA  your Vets API Terminal Production access has expired.

Since it has been more than 72 hours without a response, this is your final reminder. If we do not hear from you by the end of today, your access will be revoked and you will be required to submit a new request to regain access.
